### ISSUE TITLE
Alexa Carlson's Review Ticket

### ISSUE THREAD

**Issue by alexac54767** [nighthawkcoders/APCSP#44]
**Alexa Carlson Period 4**
[My Fastpages](https://alexac54767.github.io/Alexa-Fastpage/) (includes blogs about collaboration)
[My Video for the Python Web Server Project](https://youtu.be/awcCm8_QRPQ)
[My post on my notes for the collaboration video](https://alexac54767.github.io/Alexa-Fastpage/notes/1)
[My post about my group's collaboration for this project](https://alexac54767.github.io/Alexa-Fastpage/collaboration/1)

**Comment by safinsingh** [comment]
Very good job on this video! This video deserves a 2.7+ because it covers all the necessary content.

**Comment by AlexKumar19** [comment]
Very cool emojis and edits. Very easy to understand the little labels are very good. 2.7+

**Comment by antonyrc6** [comment]
2.7+:everything is shown and the video is well made.

**Comment by lunaiwa** [comment]
Nice edit skills showing captions, subtitles, video cuts. Nice way to think-pair-share with your teammates! All of the work needed is shown!

**Comment by avac54765** [comment]
Great job Alexa! I think you earned a 2.7+ because you collaborated with your group. You edited locally and showed how the edits are made.

**Comment by jm1021** [comment]
Great ideas.  Video making was a little hard to follow.  Perhaps have split screen on changes.  2.7

**Comment by bushku** [comment]
You deserve a 2.7+ because you included collaboration.
